app/src/common/shared/com/igalia/wolvic/ui/widgets/settings/VoiceSearchLanguageOptionsView.java

```diff
@@ -32,11 +34,17 @@ private void initialize(Context aContext) {
         updateUI();
     }
 
+    private SpeechRecognizer getSpeechRecognizer() {
+        VRBrowserApplication application = (VRBrowserApplication) getContext().getApplicationContext();
+        return application.getSpeechRecognizer();
```

Comment:
You can use a single line for this, but not a strong opinion.

Reply:
Done.